<?xml version="1.0" encoding="UTF-8"?>
<configuration>
  <property>
    <name>dfs.nameservices</name>
    <value>nameservice1</value>
  </property>
  <property>
    <name>dfs.client.failover.proxy.provider.nameservice1</name>
    <value>org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider</value>
  </property>
  <property>
    <name>dfs.ha.automatic-failover.enabled.nameservice1</name>
    <value>true</value>
  </property>
  <property>
    <name>ha.zookeeper.quorum</name>
    <value>{{ default .Env.HA_ZOOKEEPER_QUORUM "localhost:2181" }}</value>
  </property>
  <property>
    <name>dfs.ha.namenodes.nameservice1</name>
    <value>namenode1,namenode2</value>
  </property>
  <property>
    <name>dfs.namenode.rpc-address.nameservice1.namenode1</name>
    <value>{{ default .Env.DFS_NAMENODE1_RPC_ADDRESS "localhost:8020" }}</value>
  </property>
  <property>
    <name>dfs.namenode.servicerpc-address.nameservice1.namenode1</name>
    <value>{{ default .Env.DFS_NAMENODE1_SERVICERPC_ADDRESS "localhost:8022" }}</value>
  </property>
  <property>
    <name>dfs.namenode.http-address.nameservice1.namenode1</name>
    <value>{{ default .Env.DFS_NAMENODE1_HTTP_ADDRESS "localhost:50070" }}</value>
  </property>
  <property>
    <name>dfs.namenode.https-address.nameservice1.namenode1</name>
    <value>{{ default .Env.DFS_NAMENODE1_HTTPS_ADDRESS "localhost:50470" }}</value>
  </property>


  <property>
    <name>dfs.namenode.rpc-address.nameservice1.namenode2</name>
    <value>{{ default .Env.DFS_NAMENODE2_RPC_ADDRESS "localhost:8020" }}</value>
  </property>
  <property>
    <name>dfs.namenode.servicerpc-address.nameservice1.namenode2</name>
    <value>{{ default .Env.DFS_NAMENODE2_SERVICERPC_ADDRESS "localhost:8022" }}</value>
  </property>
  <property>
    <name>dfs.namenode.http-address.nameservice1.namenode2</name>
    <value>{{ default .Env.DFS_NAMENODE2_HTTP_ADDRESS "localhost:50070" }}</value>
  </property>
  <property>
    <name>dfs.namenode.https-address.nameservice1.namenode2</name>
    <value>{{ default .Env.DFS_NAMENODE2_HTTPS_ADDRESS "localhost:50470" }}</value>
  </property>
  <property>
    <name>dfs.replication</name>
    <value>3</value>
  </property>
  <property>
    <name>dfs.blocksize</name>
    <value>134217728</value>
  </property>
  <property>
    <name>dfs.client.use.datanode.hostname</name>
    <value>false</value>
  </property>
  <property>
    <name>fs.permissions.umask-mode</name>
    <value>022</value>
  </property>
  <property>
    <name>dfs.namenode.acls.enabled</name>
    <value>false</value>
  </property>
  <property>
    <name>dfs.client.read.shortcircuit</name>
    <value>false</value>
  </property>
  <property>
    <name>dfs.domain.socket.path</name>
    <value>/var/run/hdfs-sockets/dn</value>
  </property>
  <property>
    <name>dfs.client.read.shortcircuit.skip.checksum</name>
    <value>false</value>
  </property>
  <property>
    <name>dfs.client.domain.socket.data.traffic</name>
    <value>false</value>
  </property>
  <property>
    <name>dfs.datanode.hdfs-blocks-metadata.enabled</name>
    <value>true</value>
  </property>
  <property>
    <name>dfs.encryption.key.provider.uri</name>
    <value>kms://http@hdp01mn0001.s1.usw2.origami42.com:16000/kms</value>
  </property>
</configuration>
